README.md

```diff
@@ -1,2 +1,12 @@
-# HW01
-Edit the readme
+# Idongesit Ekpo's Bio for HW 1
+Hello, my name is Idongesit Ekpo and I have lived in Utah for the past **four years**. Before living in Utah, I lived in FCT Abuja, Imo and Lagos. These are all Nigerian territories and states. Living in these places has shaped my life and how I interact with people. One life lesson that I learnt from all these places is that ***everyone has a story.*** Some of my hobbies are listed below. They include:
+* Reading
+  * I am currently reading _23 Things They Didn't Tell You About Capitalism_
+* Hiking
+  * One hobby I developed as a result of living in Utah. Utah has **MOUNTAINS** everywhere. Here's a picture of Deer Valley Mountain 
+![Deer Valley](https://lifedonewell.today/wp-content/uploads/2018/06/deer-valley-mountains.jpg) 
+* Meeting new people 
+* Weight Lifting 
+* Participating in adrenaline activities.
```

Comment:
Oh my goodness can't relate xD What's the craziest thing you've done?

Reply:
The craziest thing I've done is rappelling but I haven't done much of it.